packages/core/lib/middleware/sampling/rule_cache.js

```diff
@@ -55,7 +55,7 @@ var RuleCache = {
       if(v !== 0) return v;
       if(a.getName() > b.getName())
         return 1;
```

Comment:
I'd expect our lint to enforce braces around blocks

Reply:
Good call, will add the `curly` rule